src/library/scala/util/Either.scala

```diff
@@ -526,7 +521,6 @@ object Either {
 
   /** Projects an `Either` into a `Left`.
    *
-   *  @author <a href="mailto:research@workingmouse.com">Tony Morris</a>, Workingmouse
```

Comment:
If this doesn't get merged, workingmouse.com is [only $9,888](http://workingmouse.com)!